### PR TITLE
Make SDK initialization more faster

### PR DESCRIPTION
**Pull request by andrewkochura**
Add progress bar for starting example application;
Make SDK initialization more faster.

### CODE REVIEW

**Comment by arodiss** [comment]
@andrewkochura 
please fix tests

**Comment by andrewkochura** [comment]
For @segabriel

File: example/src/main/java/com/hyber/example/AuthorizationActivity.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        // Specify the type of input expected
+        input.setInputType(InputType.TYPE_CLASS_PHONE | InputType.TYPE_TEXT_FLAG_AUTO_COMPLETE);
+        builder.setView(input);
+
+        builder.setTitle("Enter phone number");
+
+        builder.setPositiveButton("OK", new DialogInterface.OnClickListener() {
+            @Override
+            public void onClick(DialogInterface dialogInterface, int i) {
+                try {
```
Comment (OlegShevchuk) [inline example/src/main/java/com/hyber/example/AuthorizationActivity.java]: try {
                    mPhone = Long.parseLong(input.getText().toString());
                    userRegistration();
                } catch (Exception e) {
                    //logging
                }
                   
                }

File: example/src/main/java/com/hyber/example/SplashActivity.java [hunk 2]
```diff
@@ -36,89 +31,59 @@ protected void onCreate(Bundle savedInstanceState) {
...
         deviceDataUpdate();
+    }
 
-        userRegisterButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                showRegisterDialog(SplashActivity.this);
+    class ProgressTask extends AsyncTask<Integer, Integer, String> {
+        @Override
+        protected String doInBackground(Integer... params) {
```
Comment (segabriel) [inline example/src/main/java/com/hyber/example/SplashActivity.java]: You don't use params, remove it

File: example/src/main/java/com/hyber/example/SplashActivity.java [hunk 3]
```diff
@@ -36,89 +31,59 @@ protected void onCreate(Bundle savedInstanceState) {
...
+            while (mProgressStatus < maxProgress) {
+                mPrevProgressStatus = mProgressStatus;
+                if (isFinished) {
+                    mProgressStatus += maxProgressStep;
+                } else {
+                    mProgressStatus += minProgressStep;
+                    try {
+                        Thread.sleep(progressWaitMills);
+                    } catch (InterruptedException e) {
+                        e.printStackTrace();
```
Comment (OlegShevchuk) [inline example/src/main/java/com/hyber/example/SplashActivity.java]: what in this&

File: example/src/main/java/com/hyber/example/AuthorizationActivity.java (hunk 1 above)
Comment (andrewkochura) [inline example/src/main/java/com/hyber/example/AuthorizationActivity.java]: Done

File: example/src/main/java/com/hyber/example/SplashActivity.java (hunk 3 above)
Comment (andrewkochura) [inline example/src/main/java/com/hyber/example/SplashActivity.java]: Done

**Pull request merged**